string NAME=move

ObjectDrone d
ObjectLocation from
ObjectLocation to
